### PR TITLE
feat: list2and6nopic for 1366px tablet

### PR DESCRIPTION
**Pull request by apostolnikov**
[Story](https://nidigitalsolutions.jira.com/browse/REPLAT-8559)

Font size changes for tiles: 
TileAS, TileL

image

### CODE REVIEW

File: packages/edition-slices/src/tiles/tile-l/styles/index.js [hunk 1]
```diff
@@ -4,51 +4,25 @@ import {
...
-  ...sharedStyles
-};
-
-const mediumBreakpointStyles = {
-  ...sharedStyles,
-  headlineStyle: {
-    ...sharedStyles.headlineStyle,
-    fontSize: 18,
-    lineHeight: 18,
-    marginBottom: 0
```
Comment (infreezer) [inline packages/edition-slices/src/tiles/tile-l/styles/index.js]: We need to set the marginBottom to 0. For medium+ breakpoint. Not sure about mobile.
image

Comment (apostolnikov) [reply packages/edition-slices/src/tiles/tile-l/styles/index.js]: Where did you checked that?
By design, there should be margin bottom of 10px for these breakpoints.
Medium:
image
Wide:
image

Huge:
image

Comment (infreezer) [reply packages/edition-slices/src/tiles/tile-l/styles/index.js]: Currently the total spacing is 20.
image

File: packages/edition-slices/src/tiles/tile-l/styles/index.js [hunk 2]
```diff
@@ -4,51 +4,25 @@ import {
   editionBreakpoints
 } from "@times-components/styleguide";
 
-const sharedStyles = {
+const fontSizeResolver = {
+  [editionBreakpoints.small]: 18,
```
Comment (infreezer) [inline packages/edition-slices/src/tiles/tile-l/styles/index.js]: The font size on small was 20.

**Pull request merged**